src/config/job-config.ts

```diff
@@ -135,9 +135,7 @@ export default class MiraConfig {
           const envData: EnvData = loadEnvironment(envKey)
           return massageEnvData(envData, false)
         })
-        if (!config.get('cicd.buildspec_file')) {
-          throw new Error('Missing config.cicd.buildspec_file')
```

Comment:
This line will never be executed because the check is already being made on line 105